### PR TITLE
chore: add eslint on circleci

### PR DESCRIPTION
**Pull request by sanchitgera**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
     "@tophat/commitizen-adapter": "^0.0.7",
     "@tophat/commitlint-config": "^0.0.7",
+    "@tophat/eslint-config": "^0.1.4",
+    "commitizen": "^3.1.1",
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-jest": "^22.5.0",
+    "eslint-plugin-prettier": "^3.0.1",
+    "husky": "^1.3.1",
+    "lerna": "^3.13.3",
```
Comment (Yuuki-chan) [inline package.json]: This is a duplicate. This already exists in line 19.

File: package.json [hunk 2]
```diff
@@ -5,6 +5,15 @@
     "@commitlint/cli": "^7.5.2",
     "@tophat/commitizen-adapter": "^0.0.7",
     "@tophat/commitlint-config": "^0.0.7",
+    "@tophat/eslint-config": "^0.1.4",
+    "commitizen": "^3.1.1",
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-jest": "^22.5.0",
+    "eslint-plugin-prettier": "^3.0.1",
+    "husky": "^1.3.1",
```
Comment (Yuuki-chan) [inline package.json]: husky is also a duplicate here

File: package.json [hunk 3]
```diff
@@ -5,6 +5,15 @@
     "@commitlint/cli": "^7.5.2",
     "@tophat/commitizen-adapter": "^0.0.7",
     "@tophat/commitlint-config": "^0.0.7",
+    "@tophat/eslint-config": "^0.1.4",
+    "commitizen": "^3.1.1",
```
Comment (Yuuki-chan) [inline package.json]: also a duplicate

File: package.json [hunk 4]
```diff
@@ -14,6 +23,10 @@
       "commit-msg": "commitlint -e $GIT_PARAMS"
     }
   },
+  "scripts": {
+    "lint": "eslint --ignore-path .gitignore .",
+    "lint-fix": "eslint --ignore-path .gitignore . --fix"
```
Comment (Yuuki-chan) [inline package.json]: Are these scripts used anywhere?

Comment (sanchitgera) [reply package.json]: These scripts are used by circleci for linting so we don't have to install eslint globally

**Pull request merged**